app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -143,4 +109,39 @@ private void splitEventsAndAddToAdapter(Collection<? extends SimpleEvent> events
         Collections.reverse(past);
         return new Pair<>(upcoming, past);
     }
+
+    @Override
+    protected boolean onListLoadMore(int page, int totalItemsCount) {
+        App.getInstance().getGdgXHub().getChapterAllEventList(
+            mPlusId, page).
+            enqueue(new Callback<PagedList<Event>>() {
+                @Override
+                public void success(PagedList<Event> eventsPagedList) {
+                    List<Event> events = eventsPagedList.getItems();
+                    mAdapter.addAll(events);
+                    App.getInstance().getModelCache().putAsync(mCacheKey,
+                        mEvents,
+                        DateTime.now().plusHours(2),
+                        new ModelCache.CachePutListener() {
+                            @Override
+                            public void onPutIntoCache() {
+                                mAdapter.addAll(mEvents);
+                                setIsLoading(false);
+                            }
+                        });
+                }
+
+                @Override
+                public void failure(Throwable error) {
+                    onError(R.string.fetch_events_failed);
+                }
+
+                @Override
+                public void networkFailure(Throwable error) {
+                    onError(R.string.offline_alert);
+                }
+            }
+        );
+        return true;
```

Comment:
We are always returning true here? How do we stop loading? When we are at the end of the data, we should stop loading more pages.